pages/apis/rest_api/artifacts.md.erb

```diff
@@ -150,3 +150,17 @@ curl "https://api.buildkite.com/v2/organizations/{org.slug}/pipelines/{pipeline.
 Required scope: `read_artifacts`
 
 Success response: `302 Found`
+
+## Delete an artifact
+
+Marks the artifact as deleted, removes it from the job and builds artifact list, and removes the file from Buildkite’s AWS S3 artifacts bucket.
+
+If the artifact was stored in your own [AWS S3](https://buildkite.com/docs/agent/v3/cli-artifact#using-your-own-private-aws-s3-bucket) or [Google Cloud](https://buildkite.com/docs/agent/v3/cli-artifact#using-your-own-private-google-cloud-bucket) storage, it will not be automatically deleted from your store.
```

Comment:
What happens in this case? It's still removed from the lists, but not actually deleted from their S3?

Reply:
Yep, that’s it.